### PR TITLE
READY: Removing pagerank

### PR DESCRIPTION
**Pull request by vandenheuvel**
This pull request remove the page rank algorithm in it's entirety from the display.  Node color is now based upon the difference in total up and download for a node.

### CODE REVIEW

File: TriblerGUI/widgets/trustpage/js/radial_view/radial_node.js [hunk 1]
```diff
@@ -194,13 +195,12 @@ function RadialNodes(svg, options) {
...
             .range(self.config.color.range);
 
-        // Map relative to the minimum and maximum page rank of the graph
-        var rank_difference = self.graphData.max_page_rank - self.graphData.min_page_rank;
-        if (rank_difference === 0) {
-            return nodeColor(1);
-        }
+        var difference = node.total_up - node.total_down;
+        difference = Math.max(difference, self.config.upDownDifferenceDomain.min);
+        difference = Math.min(difference, self.config.upDownDifferenceDomain.max);
```
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/radial_view/radial_node.js]: Maybe consider putting this in one line: 
```javascript
Math.min(
    Math.max(node.total_up - node.total_down, self.config.upDownDifferenceDomain.min),
    self.config.upDownDifferenceDomain.max
)
```

File: TriblerGUI/widgets/trustpage/js/request_data.js [hunk 2]
```diff
@@ -49,7 +48,7 @@ function make_cors_request(method, url) {
  * @param callback The callback which is called with the GraphResponseData
  */
 function get_node_info(public_key, neighbor_level, callback) {
-    var url = "http://localhost:8085/multichain/network?focus_node=" + public_key + "&neighbor_level=" + neighbor_level;
+    var url = "http://localhost:8085/multichain/network?focus_node=" + public_key + "&neighbor_level=";
```
Comment (yinghaodai) [inline TriblerGUI/widgets/trustpage/js/request_data.js]: Please undo this change
